### PR TITLE
Miscellaneous Documentation Fixes

### PR DESCRIPTION
**Pull request by kkeirstead**
Noticed a few broken links and typos when reading through docs.

### CODE REVIEW

File: docs/troubleshooting.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ See [CLRIE in Azure](scenarios/azure.md).
 
 ## Mitigation
 
-When an issue occurs in production, the first step is to mitigate it so the application continues to run. A common cause of problems are from updates to either CLRIE or to one of the Instrumentation Method client that introduces potential conflicts with another client or incompatiblities with the app.
+When an issue occurs in production, the first step is to mitigate it so the application continues to run. A common cause of problems are from updates to either CLRIE or to one of the Instrumentation Method clients that introduces potential conflicts with another client or incompatiblities with the app.
```
Comment (schmittjoseph) [inline docs/troubleshooting.md:9]:
```suggestion
When an issue occurs in production, the first step is to mitigate it so the application continues to run. A common cause of problems are from updates to either CLRIE or to one of the Instrumentation Method clients that introduces potential conflicts with another client or incompatibilities with the app.
```

**Review by schmittjoseph — COMMENTED** [review]
Thanks for cleaning this up!

**Comment by WilliamXieMSFT** [comment]
+1

---
In reply to: [894300613](https://github.com/microsoft/CLRInstrumentationEngine/pull/469#issuecomment-894300613) [](http://example.com/codeflow?ancestors=894300613,894300613)

**Review by WilliamXieMSFT — APPROVED** [review]
:shipit:

**Pull request merged**